vars/buildNodeJs.groovy

```diff
@@ -0,0 +1,95 @@
+def call(Map config=[:], Closure body={}) {
+  def containerSrcFolder = '\\/home\\/node'
```

Comment:
Would it make sense to declare the variables outside of the function itself or is there a benefit this provides?

Reply:
The way I've read how Groovy files get interpreted is that variables outside of a function get incorporated into a 'main' function, thus aren't accessible to any other functions in the file. Neil and I tested this a month or so back and it did indeed seem to be the case.

Reply:
Here's an example, I ran this at https://groovyconsole.appspot.com:
```

​def myvar = 'abc'

def myFunc() {
    println myvar;
}

myFunc()
​```
When executed, it throws an exception in myFunc: `groovy.lang.MissingPropertyException: No such property: myvar for class: Script1`